projects/core/src/util/command-query/command.service.ts

```diff
@@ -53,6 +54,16 @@ export class CommandService implements OnDestroy {
     const commands$ = new Subject<PARAMS>();
     const results$ = new Subject<ReplaySubject<RESULT>>();
 
+    // We have to provide specify handlers after RxJS version 7.3.0
```

Comment:
[NITPICK]
```suggestion
    // We have to specify handlers explicitly after RxJS version 7.3.0,
    // Otherwise, the `unsubscribe` handler would be passed implicitly to `notifier$` when calling `tap(notifier$)`
```